crates/metal-ungram/src/error.rs

```diff
@@ -10,6 +10,6 @@ pub enum Error {
     Io(#[from] std::io::Error),
     #[error("Ungrammar error: {0}")]
     Ungram(#[from] ungrammar::Error),
-    #[error("`metal-ast-ungram` is expected to be run from build.rs of metal-ast-ng only.")]
+    #[error("`metal-ast-ungram` is expected to be run from build scripts only.")]
```

Comment:
```suggestion
    #[error("`metal-ungram` is expected to be run from build scripts only.")]
```

Reply:
Oh, thanks, that's weird. I have a memory of updating this